server/neptune/workflows/internal/config/logger/logger.go

```diff
@@ -0,0 +1,34 @@
+package logger
+
+import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/config"
+	"go.temporal.io/sdk/workflow"
+)
+
+func Info(ctx workflow.Context, msg string) {
```

Comment:
The old atlantis logger used to support a format like this and after seeing that I'm pretty opinionated on forcing structured logging and making it hard to do formatted logging so that it's not abused.  

We can support additional kvs though if necessary, there's no current examples of this though so we can add it when we need it.

Reply:
> I'm pretty opinionated on forcing structured logging and making it hard to do formatted logging so that it's not abused.

I'd prefer that over both the current implementation and even my suggestion, but we'd need another different API to make it happen. While I think this thread is worth discussing, I realize it's gonna be a significant enough tangent from the PR's intent itself that we should probably discuss it separately. 

As for current examples—there are multiple places in this PR where we're formatting a message in a way where we could spawn off the variable portion into additional structured k/vs.